src/annotator/integrations/vitalsource.js

```diff
@@ -142,13 +178,38 @@ export class VitalSourceContentIntegration {
         e.stopPropagation();
       });
     }
+
+    // If this is a PDF, create the hidden text layer above the rendered PDF
+    // image.
+    const bookImage = document.querySelector('#pbk-page');
+    const pageData = getPDFPageData();
+    if (bookImage && pageData) {
+      this._textLayer = new ImageTextLayer(
+        bookImage,
+        pageData.glyphs.glyphs.map(glyph => ({
+          left: glyph.l / 100,
+          right: glyph.r / 100,
+          top: glyph.t / 100,
+          bottom: glyph.b / 100,
```

Comment:
Would it be better to do this transformation within the `ImageTextLayer`?

Reply:
`ImageTextLayer` uses generic coordinates from 0-1. Specific integrations may use different coordinate systems in the bounding box data they provide to us. For example they may use:

- Percentages (as VS does)
- Pixels
- The PDF page coordinate system